tests/integration/protostar_toml/protostar_toml_test.py

```diff
@@ -0,0 +1,20 @@
+from pathlib import Path
+
+from pytest_mock import MockerFixture
+
+from protostar.protostar_toml.protostar_toml_reader import ProtostarTOMLReader
+from protostar.protostar_toml.protostar_toml_writer import ProtostarTOMLWriter
+from protostar.utils.protostar_directory import VersionManager
+
+
+def test_saving_and_reading(tmpdir, mocker: MockerFixture):
+    version_manager_mock = mocker.MagicMock()
```

Comment:
Wouldn't be better if this test loaded each section using `from_protoster_reader_toml` and check values in section object?

Reply:
It would be.